website/client/components/guilds/groupFormModal.vue

```diff
@@ -0,0 +1,285 @@
+<template lang="pug">
+  b-modal#modal1(:title="$t('createGuild')", :hide-footer="true")
```

Comment:
is the id necessary? in which case it's probably a good idea to use a descriptive name